### ISSUE TITLE
Someone should use this pic somewhere.

### ISSUE THREAD

**Issue by jwietelmann** [joeellis/barcamp2012#22]
![gorbyjohn](http://i.imgur.com/Ky1Pi.png)

Just sayin'...

**Comment by joeellis** [comment]
Good lord, how is this not already in a PR?

**Comment by joeellis** [comment]
Maybe this can be @toots work if he's not already done

**Comment by joeellis** [comment]
Or even @krainboltgreene since I know he loves that meme of his

**Comment by toots** [comment]
I'd love to take care of it but I have no idea for now..

**Comment by joeellis** [comment]
Yeah, it'd be nice to do something with it other than just stick it in one of the windows.  If someone has an idea, I don't mind taking some minutes and cutting the cat-man-thing out into a separate transparent png

**Comment by jwietelmann** [comment]
This may be overly ambitious unless the right person has enough time, but I was thinking it would be cool to take 3 windows and turn them into a slot machine. If we did that, this would be a cool way to make use of the pic:

3 raptors --> raptorizer pops up.
3 gorbys --> random pic of gorby from the tumblr account
3 john barnettes --> random tweet from john
2 johns and a gorby --> gorbyjohn

Something like that.

**Comment by toots** [comment]
I'll see what I can do :-)

**Comment by joeellis** [comment]
@toots Do you need me to cut gorbyjohn out of the picture with photoshop?

**Comment by mshwery** [comment]
Joel. had the same slot machine idea with the windows. I was just about to implement today and then I saw this. 

something like: http://odhyan.com/blog/2011/05/slot-machine-in-javascript/
or: http://matthewlein.com/experiments/slots.html

**Comment by jwietelmann** [comment]
@mshwery Have at it. I'm focused on Yippayap at the moment.
